### DISCUSSION TITLE
Possible LDAP Sync issue - Active Directory

### DISCUSSION THREAD

**Discussion by DaveDischord** [modoboa/modoboa#2080] [Help]
# Impacted versions

* OS Type: Debian/Ubuntu
* OS Version: 18.04
* Database Type: PostgreSQL
* Database version: 10.14
* Modoboa: 1.16.1
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
Connect to server via Outlook or Webmail

# Current behavior
Issue arose after updating Modoboa from 1.15 to 1.16
Certain users (no pattern found) cannot log on using their AD password, but others can, even after resetting password and giving server >1 hour to "sync up".

Edit:  Possibly unrelated issue: Even when set to 3 minute fetch, pulling mail (via Outlook) seems slow.

# Expected behavior
Current AD Password works to log into Modoboa webmail and/or by using Outlook
 
# Video/Screenshot link (optional)

**Comment by tonioo** [comment]
@DaveDischord DO you have the possibility to get some logs about authentication on your AD server?

**Comment by DaveDischord** [comment]
> @DaveDischord DO you have the possibility to get some logs about authentication on your AD server?

Any suggestions on where to look (like where in Event Viewer).? I don't have any third-party tools installed.

**Comment by tonioo** [comment]
@DaveDischord Could you update your logging configuration like this (in the settings.py file):
```python
LOGGING = {
    'version': 1,
    'formatters': {
        'syslog': {
            'format': '%(name)s: %(levelname)s %(message)s'
        },
    },
    'handlers': {
        'mail-admins': {
            'level': 'ERROR',
            'class': 'django.utils.log.AdminEmailHandler',
            'include_html': True
        },
        'syslog-auth': {
            'class': 'logging.handlers.SysLogHandler',
            'facility': SysLogHandler.LOG_AUTH,
            'formatter': 'syslog'
        },
        'modoboa': {
            'class': 'modoboa.core.loggers.SQLHandler',
        },
        'console': {
            'class': 'logging.StreamHandler'
        }
    },
    'loggers': {
        'django': {
            'handlers': ['mail-admins'],
            'level': 'ERROR',
            'propagate': False
        },
        'modoboa.auth': {
            'handlers': ['syslog-auth', 'modoboa'],
            'level': 'INFO',
            'propagate': False
        },
        'modoboa.admin': {
            'handlers': ['modoboa'],
            'level': 'INFO',
            'propagate': False
        },
        'django_auth_ldap': {
            'level': 'DEBUG',
            'handlers': ['console', 'syslog-auth']
        },
    }
}
```
Make sure to reload uwsgi after. You should see more information in your auth log file

**Comment by DaveDischord** [comment]
@tonioo Will do.  In the settings.py, there is **DOVECOT_LOOKUP_PATH** commented out.  Is that relevant to AD LADAP, since I have the dovecot file configured?  Asking out of curiosity since I haven't noticed it before, or seen it referenced.
